submissions/emmanuelEsuong/javaScript/signUp.js

```diff
@@ -0,0 +1,177 @@
+let submitForm = document.getElementById('validateForm'),
+    firstName = document.getElementById('firstName'),
+    lastName = document.getElementById('lastName'),
+    email = document.getElementById('email'),
+    number = document.getElementById('number'),
+    passWord = document.getElementById('password'),
+    confirmPassword = document.getElementById('confirmPassword'),
+    terms = document.getElementById('terms')
+    ;
+
+    //required fields function
+     function required(field, event) {
+        if (field.value === '') {
+            field.nextElementSibling.innerHTML = "This field is required";
+            event.preventDefault();
+            return false;
+        }   else {
+            field.nextElementSibling.innerHTML = "";
+            return true;
+        }
+    }
+
+    //name fields validation regex
+    function lettersOnly(firstName, lastName, otherName) {
+        //validate name fields to accepted only letters/cass sensitive
+        var regex = /[^a-z]/gi;
+        firstName.value = firstName.value.replace(regex, "");
+        lastName.value = lastName.value.replace(regex, "");
+        otherName.value = lastName.value.replace(regex, "");
+    }
+    //firstName field validation conditions
+    function firstNameFieldValidation(event) {
+        //prevent empty name field submission
+        if(firstName.value === '') {
```

Comment:
You already wrote a function that checks if a field is empty. There is no need for this check